stackrox/stackrox
# ROX-21132: Redirect from integrations to cluster init bundles route

## Description

1. Add conditional rendering to integrations route that corresponds to tile in #9348
2. Delete conditional rendering for user role permissions that is superseded by #7128

## Checklist

## Testing Performed

Before you `yarn deploy` in ui folder:

```sh
export ROX_MOVE_INIT_BUNDLES_UI=true
```

1. `yarn lint` in ui/apps/platform
2. `yarn build` in ui/apps/platform
3. `yarn start` in ui

### Manual testing

1. Enter in browser address bar and then press Enter:

    * /main/integrations/authProviders/clusterInitBundle
    * /main/integrations/authProviders/clusterInitBundle/view/id

    See redirect to /main/clusters/init-bundles

### Integration testing

Before you `yarn cypress-open` in ui/apps/platform folder:

```sh
export CYPRESS_ROX_MOVE_INIT_BUNDLES_UI=true
```

By the way, when I forgot, the redirect worked, and then test failed because of assertions for integrations form.

* integrations/apiTokens.test.js
* integrations/clusterInitBundles.test.js skip because of feature flag
* integrations/externalBackups.test.js
* integrations/general.test.js skip one test because of feature flag
* integrations/imageIntegrations.test.js
* integrations/notifiers.test.js
* integrations/signatureIntegrations.test.js
